buildchain/buildchain/salt_tree.py

```diff
@@ -227,6 +227,7 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/addons/prometheus-operator/deployed/init.sls'),
     Path('salt/metalk8s/addons/prometheus-operator/deployed/namespace.sls'),
     Path('salt/metalk8s/addons/prometheus-operator/deployed/storageclass.sls'),
```

Comment:
Please run full test manually or from the CI, here you do not include the post upgrade and downgrade files so, i'm pretty sure it does not work 

salt/metalk8s/orchestrate/downgrade/post.sls
salt/metalk8s/orchestrate/upgrade/post.sls

Reply:
Yeah, my bad!

I missed this after doing some cleanup onthe PR